recipe/meta.yaml

```diff
@@ -11,29 +11,42 @@ source:
   patches:
     - 0001-Don-t-add-dashes-to-version.patch
     - 0001-Patch-pocketfft-for-C-17-osx-64.patch  # [osx and x86_64]
+    - 0002-fix-bazelrc-112.diff  # [cuda_compiler_version == "11.2"]
+    - 0002-fix-bazelrc-111.diff  # [cuda_compiler_version == "11.1"]
+    - 0002-fix-bazelrc-110.diff  # [cuda_compiler_version == "11.0"]
+    - 0002-fix-bazelrc-102.diff  # [cuda_compiler_version == "10.2"]
```

Comment:
@ngam and @xhochy do we really want all those cuda compiler versions? The matrix is gigantic and I wonder if it is worth maintaining all these patches. I'm biased b/c I rarely use it so I appreciate someone who has use case for this commenting here.

Reply:
I dropped all but 11.2 in the other PR. Both 11.0 and 10.2 are likely *NOT* gonna work based on feedback from upstream. Even if they pass our CI, they will likely not work correctly. 

So imho, only 11.2 is the one we should take forward. Note, this patch is sloppy because I didn't think it through, and there is a chance upstream will fix the problem causing this patch at some point soon... so I wouldn't worry about the patches per se, but just the uselessness of anything but 11.2 is somewhat obvious in this case

Reply:
@ngam curious to hear your thoughts, what if we only released 11.2-compatible jaxlib this time and added others later? (Or not even bother?) Was it a lot of work to get these patches in? (I'm asking from a position of trying to minimize labor on your end, as I know these things take time.)

Reply:
> @ngam curious to hear your thoughts, what if we only released 11.2-compatible jaxlib this time and added others later? (Or not even bother?) Was it a lot of work to get these patches in? (I'm asking from a position of trying to minimize labor on your end, as I know these things take time.)

I'd vote releasing only 11.2 and not bothering with the rest. The work isn't the deciding factor though, it would ~ same, but the bigger issue here is the actual compatibility as dictated by upstream imo. 

Also, this is something for the maintainers here and the core team to decide (I've personally long advocated for only releasing 11.2, but there is a worry about that ending disenfranchising people with older GPUs, etc.)